packages/orchestrator-ui-components/src/components/WfoTree/treeUtils.ts

```diff
@@ -32,3 +41,38 @@ export const sortTreeBlockByLabel = (
     }
     return 0;
 };
+
+const countChildeNodes = (node: TreeBlock): number => {
+    let count = 0;
+    for (const child of node.children) {
+        count += 1 + countChildeNodes(child);
+    }
+    return count;
+};
+
+export const getPositionInTree = (
+    tree: TreeBlock,
+    id: number,
+): number | undefined => {
+    let startPosition = 1;
```

Comment:
A recursive solution would earn you bonus points and would get rid of the 'let' ;)

Reply:
Damn.. missed points